src/components/modal/modal-end-chat.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import '../log-in/log-in.scss';
+import { Link } from "react-router-dom";
+
+class ModalEndChat extends Component {
+  state = {
+    feedback: 0,
+    // questionid: this.props.modalRef.questionid,
+    // expiration: Date.now() + 30
+  }
+
+  closesQuestion = () => {
+    const token = localStorage.getItem('token');
+    const body = {
+      "karma": this.state.feedback,
+      "credits": 50
+    }
+
+    fetch(`http://localhost:4000/questions/${this.props.questionId}/feedback`, {
+      method: 'PUT', 
+      headers: { 
+        'Authorization' : 'Bearer ' + token,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(body)
+    })
+      .then(res => res.json())
+      .then(res=> console.log('question CLOSED', res))
```

Comment:
Left a task in the backlog - we should dispatch an action to the the redux store with the newly earned karma of the tutor so that the user data in the navbar (which pulls the karma and credits from the store) is updated.